lib/active_file/hash_and_array_files.rb

```diff
@@ -13,7 +13,7 @@ def data_from_multiple_files
       loaded_files = full_paths.collect { |path| load_path(path) }
 
       if loaded_files.all?{ |file_data| file_data.is_a?(Array) }
-        loaded_files.sum
```

Comment:
is this something we can fix in the definition of `sum`?

Or is this just a place where the method signature has been messed up in upstream rails?